src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md

```diff
@@ -107,7 +107,7 @@ CA2101 | Globalization | Info | PInvokeDiagnosticAnalyzer, [Documentation](https
 CA2109 | Security | Disabled | ReviewVisibleEventHandlersAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2109)
 CA2119 | Security | Disabled | SealMethodsThatSatisfyPrivateInterfacesAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2119)
 CA2153 | Security | Disabled | DoNotCatchCorruptedStateExceptionsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2153)
-CA2200 | Usage | Info | RethrowToPreserveStackDetailsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2200)
```

Comment:
We should actually move the entire contents of this file back to Unshipped.md (except the `## Release 2.9.8 line) as we haven't shipped a NuGet package for Microsoft.CodeAnalysis.NetAnalyzers. I had moved it to shipped to just test warning wave support. I will do this as a follow-up PR.